src/pages/Combat.tsx

```diff
@@ -121,43 +138,55 @@ export default function CombatPage({stats, onChange, onRoll, locked} : CombatPag
             </Initiative>
         </Grid>
         <Grid item xs={6}>
-
-                <Collection
-                    heft={calculateHeft(stats)}
-                    locked={locked}
-                    talents={stats.talents}
-                    values={characterWeapons}
-                    onChange={changeWeapons}
-                    onAction={performAction}
-                    component={Weapon} />
-
+            <Grid item xs={6} marginY={1}>Weapons</Grid>
```

Comment:
You could also use `<h2>` or `<h3>` here to make the title of Weapons and Armors more visible to the user.